### PR TITLE
Impromment/nav bar movile view

### PR DESCRIPTION
**Pull request by caballerof09**
some new adjustment 
- css reorganized ,search form modified ,small improvement
- small changes navbar border ,added folder img & search icons
- minor changes sear form

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -24,14 +24,20 @@
...
+}
+
+#navbardrop {
+    outline: none; 
+    border: none; 
+    box-shadow: none;
 }
 
-h5 {
+.text-header {
```
Comment (Manuel-Suarez-Abascal) [inline css/style.css]: I'm not sure if this is a **Bootstrap 4's class**. If not, maybe you should change to `header-text` & move it up to `header` CSS declarations closer to `header-tittle-brand` classes?

File: index.html [hunk 2]
```diff
@@ -12,18 +12,20 @@
...
+
         <!-- Brand -->
         <a class="navbar-brand" href="#"><span>Deporte Rey</span></a>
         <!-- Brand and toggle get grouped for better mobile display -->
-        <button class="navbar-toggler custom-focus" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <button class="navbar-toggler mr-2 custom-focus" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
-        <!--conainer fluid-->
+        <!-- conainer fluid -->
```
Comment (Manuel-Suarez-Abascal) [inline index.html]: ` <!-- conainer fluid -->` you made a typo in comment. It should be ` <!-- container fluid -->`

**Review by Manuel-Suarez-Abascal — APPROVED** [review]
@caballerof09 Looks good to me! (LGTM) 👍 👏

File: index.html (hunk 2 above)
Comment (caballerof09) [reply index.html]: this small detail has been solve

**Comment by caballerof09** [comment]
after reviewing all the comment you let me and fixing them, i will continue with merging the branches.

**Pull request merged**